arches_lingo/src/arches_lingo/components/generic/LabelViewer.vue

```diff
@@ -0,0 +1,164 @@
+                        .appellative_status_ascribed_name_content
+                }}
+            </template>
+        </Column>
+        <Column
+            field="appellative_status_ascribed_relation"
+            :header="$gettext('Label Type')"
+            sortable
+        >
+            <template #body="slotProps">
+                <ControlledListItem
+                    :value="
+                        (slotProps.data as AppellativeStatus)
+                            .appellative_status_ascribed_relation
+                    "
+                >
+                </ControlledListItem>
+            </template>
+        </Column>
+        <Column
+            field="appellative_status_ascribed_name_language"
+            :header="$gettext('Label Language')"
+            sortable
+        >
+            <template #body="slotProps">
+                <ControlledListItem
+                    :value="
+                        (slotProps.data as AppellativeStatus)
+                            .appellative_status_ascribed_name_language
+                    "
+                >
+                </ControlledListItem>
+            </template>
+        </Column>
+        <Column>
+            <template #body="slotProps">
+                <div class="controls">
+                    <Button
+                        icon="pi pi-file-edit"
+                        :aria-label="$gettext('edit')"
+                        @click="
+                            () =>
+                                emits(
+                                    'editLabel',
+                                    (slotProps.data as AppellativeStatus)
+                                        .tileid,
+                                )
+                        "
+                    />
+                    <Button
+                        icon="pi pi-trash"
+                        :aria-label="$gettext('delete')"
+                        severity="danger"
+                        outlined
+                        @click="
+                            () =>
+                                confirmDelete(
+                                    (slotProps.data as AppellativeStatus)
+                                        .tileid,
+                                )
+                        "
+                    />
+                </div>
+            </template>
+        </Column>
+        <template #expansion="slotProps">
+            <div class="drawer">
+                <div>
+                    {{ $gettext("Bibliographic Sources") }}:
+                    <ResourceInstanceRelationships
+                        :value="
+                            (slotProps.data as AppellativeStatus)
+                                .appellative_status_data_assignment_object_used
+                        "
+                    ></ResourceInstanceRelationships>
+                </div>
+                <div>
+                    {{ $gettext("Contributors") }}:
```

Comment:
"naked" strings in HTML here too

Reply:
this file shouldn't exist anymore - this is just a rebase issue.  Fixing.